ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/AddItemsToPortal/AddItemsToPortal.cpp

```diff
@@ -203,37 +232,20 @@ void AddItemsToPortal::connectUserSignals()
       }
     }
   });
+}
 
-  //! [PortalUser addPortalItemCompleted]
```

Comment:
Doc build tags removed. If this is unused it is fine, but please confirm. If there are references to this still then we need to keep it.